source/_integrations/opentherm_gw.markdown

```diff
@@ -43,7 +43,7 @@ The following configuration options are available:
 name:
   description: "The friendly name used for the OpenTherm Gateway and its entities. This is used to generate the display name of the created device and all related entities.<br/>Examples: `Thermostat`, `Living Room`"
 path or url:
-  description: "Path to the OpenTherm Gateway device as supported by [PySerial](https://pythonhosted.org/pyserial/url_handlers.html). This is usually either a path to a serial device in `/dev/` if the gateway is connected via serial or USB or a URL in the form of `socket://[IP address]:[port]` if it is connected over the network.<br/>Examples: `/dev/ttyUSB0`, `socket://192.168.0.250:7686`"
+  description: "Path to the OpenTherm Gateway device as supported by [PySerial](https://pythonhosted.org/pyserial/url_handlers.html). This is usually either a path to a serial device in `/dev/` if the gateway is connected via serial or USB or a URL in the form of `socket://[IP address]:[port]` if it is connected over the network.<br/>Examples: `/dev/ttyUSB0`, `socket://192.168.0.250:7686`". Default port for the current NodeMCU firmware is 25238 as it's mentioned in the [documentation](https://www.otgw.tclcode.com/otgwmcu.html). For older versions of the ESPHome firmware, you could choose port number in the device's settings via Web interface (e.g. `6638`). 
```

Comment:
Adding a "default" port number has been suggested before, but I don't think this is the right place to document it.

The OpenTherm Gateway was originally designed as a serial device. Any method used to connect via the network is using a separate device (usually an ESP-based wifi controller, but wired options are also available) to send the serial data to the network and vice versa. There are many serial-to-network options available and especially the ESP-based options can be flashed with a lot of different firmwares, each with their own default port.

Because of this, the "default" port is dependent on what people are using to connect to their gateway through the network. In my opinion the right place to document this default is the documentation of the firmware they are using.
In the case of the otgwmcu firmware it is documented at the bottom of [this page](https://otgw.tclcode.com/otgwmcu.html)

Reply:
Fair enough, I will remove this block. I intended to show some default numbers if people got a ready-to-use device like the one from Nodo Shop so they won't spend time looking for them.